webpack/scenes/ContentViews/Details/ComponentContentViews/ComponentVersion.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+import { translate as __ } from 'foremanReact/common/I18n';
+import {
+  Text,
+  TextContent,
+  TextVariants,
+} from '@patternfly/react-core';
+
+const ComponentVersion = ({ componentCV }) => {
+  const { latest, content_view_version: cvVersion, content_view: cv } = componentCV;
+  const {
+    id,
+    latest_version: latestVersion,
+  } = cv;
+  const { version } = cvVersion;
+  const latestDescription = __('Latest(automatically updates)');
+  const manualVersionText = (latestVersion === version) ? __('Latest Version') : __(`New Version is available: Version ${latestVersion}`);
+  return (
+    <React.Fragment>
```

Comment:
Pretty sure we can just use `<> ... </>` instead of `<React.Fragment> ... </React.Fragment>` now! It should work as long as you don't need to assign a `key` prop.